tests/Command/ValidatorsTest.php

```diff
@@ -62,6 +69,10 @@ public function getValidateEntityNameTests(): array
     }
 
     /**
+     * NEXT_MAJOR: Remove this test.
+     *
+     * @group legacy
+     *
      * @dataProvider getValidateEntityNamesWithExceptionTests
```

Comment:
This `getValidateEntityNamesWithExceptionTests()` needs to be removed as well in the next major

Reply:
Good catch.